extension/WebJobs.Extensions.RabbitMQ.Samples/WebJobs.Extensions.RabbitMQ.Samples.csproj

```diff
@@ -6,6 +6,8 @@
     <LangVersion>latest</LangVersion>
     <AssemblyName>Microsoft.Azure.WebJobs.Extensions.RabbitMQ.Samples</AssemblyName>
     <RootNamespace>Microsoft.Azure.WebJobs.Extensions.RabbitMQ.Samples</RootNamespace>
+    <IsPackable>false</IsPackable>
+    <EnableNETAnalyzers>false</EnableNETAnalyzers>
```

Comment:
The test and sample projects are targeting .NET 6. If .editorconfig is found in the repo, the .NET analyzers get kicked in without any additional steps. For now, I need to disable the analysis, else otherwise the build will fail due to codeanalysis errors.